api/loader/getter.go

```diff
@@ -58,32 +58,41 @@ func newLoaderAtGetter(raw string, fSys filesys.FileSystem, referrer *fileLoader
 func getRemoteTarget(rs *remoteTargetSpec) error {
 	var err error
 
-	rs.TempDir, err = filesys.NewTmpConfirmedDir()
+	// Get the pwd
+	pwd, err := os.Getwd()
 	if err != nil {
-		return err
+		log.Fatalf("Error getting wd: %s", err)
```

Comment:
This isn't actually a change, I made. The original source panics here, I assume because we expect `os.Getwd()` to always be successful. Should I still just go make this return an error?